src/foam/classloader/NodeModelFileDAO.js

```diff
@@ -23,20 +23,40 @@ foam.CLASS({
   properties: [
     {
       name: 'classpath'
+    },
+    {
+      name: 'loadedModels',
```

Comment:
That's contextualized. `require()` is global. Re-`arequire`-ing in different contexts is what instigated this change.